onnxruntime/python/tools/kernel_explorer/kernels/skip_layer_norm.cc

```diff
@@ -123,6 +147,16 @@ class SkipLayerNormTunable : public IKernelExplorer {
       .def("Run", &SkipLayerNormTunable<type>::Run)                            \
       .def("IsSupported", &SkipLayerNormTunable<type>::IsSupported);
 
+#define REGISTER_ORIGINAL_OP(type)                                               \
```

Comment:
This is largely the same as `REGISTER_TUNABLE_OP`, can we make `REGISTER_TUNABLE_OP` a more generic macro to avoid adding a new macro?